### PR TITLE
Update leadership_team.md

### PR DESCRIPTION
**Pull request by afromme**
removing ED

### CODE REVIEW

**Review by jimentwood — APPROVED** [review]
Looks good. I modified people/staff as well

**Review by bmaltzan — APPROVED** [review]

**Review by SBBCornell — COMMENTED** [review]
with Eleonora removed, the top row will only have Steinn. Is that intentional?

File: about/people/leadership_team.md [hunk 1]
```diff
@@ -25,7 +19,7 @@ people:
     position: Community Engagement and Development Coordinator
     thumbnail: images/alison.jpg
     bio: |
-      Alison coordinates arXiv's communications and fundraising efforts, including the membership program for academic and research institutions. Alison's background is in science writing, nonprofit fundraising, and teaching. She is a Cornell University alumn and holds an M.S. in Zoology from Washington State University.
+      Alison leads arXiv's communications and fundraising efforts, including the membership program for academic and research institutions. Alison's background is in science writing, nonprofit fundraising, and teaching. She is a Cornell University alumn and holds an M.S. in Zoology from Washington State University.
```
Comment (helendwang) [inline about/people/leadership_team.md:22]: Think it should read "Cornell University alumna" or "alum"

**Review by mhl10 — APPROVED** [review]

**Comment by afromme** [comment]
I rearranged the order of appearance on the page, in response to Shamsi's comment

**Pull request merged**